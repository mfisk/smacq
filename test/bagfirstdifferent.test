print -v srcip from (pcapfile(0325@1112-snort.pcap) + tabularinput(-d " " -f maillogjoin.data 1 2 3 srcip))
