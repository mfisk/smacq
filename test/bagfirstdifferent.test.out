srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.200
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = 192.168.0.54
srcip = mailrelay2
srcip = mailrelay2
srcip = mailrelay2
srcip = mailrelay2
srcip = mailrelay2
srcip = mailrelay2
srcip = mailrelay2
srcip = mailrelay2
srcip = mailproxy2
srcip = mailrelay2
srcip = mailrelay3
srcip = mailrelay3
srcip = mailrelay3
srcip = mailrelay3
srcip = mailrelay3
srcip = mailproxy2
srcip = mailrelay3
